src/main/resources/security-config.yml

```
@@ -45,3 +45,7 @@ pbkdf2:
   secret: ${PBKDF2_SECRET}
   saltLength: ${PBKDF2_SALT_LENGTH}
   iterations: ${PBKDF2_ITERATIONS}
+
+login:
+  max-attempt: 5
+  lock-time-seconds: 60000 # 5 minutes
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)